spec/geogem_spec.rb

```diff
@@ -49,7 +49,15 @@
     it 'returns true if 2 line segments intersect' do
       my_obj = Polygon.new("0 0")
       expect(my_obj.send(:intersect?, [Point.new("0 0"), Point.new("2 2")], [Point.new("2 0"), Point.new("0 2")]))
-      .to eq (true)
+      .to eq(true)
+    end
+  end
+
+  describe '#intersect?' do
```

Comment:
To be honest you're correct here. I didn't realise from my first read through that you were testing a method you'd build yourself instead of just using a library! As you were :)